src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2199,6 +2199,85 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
+                                ->scalarNode('passphrase')
+                                    ->info('A passphrase of the private key if any')
+                                    ->defaultValue('')
+                                ->end()
+                                ->arrayNode('options')
+                                    ->performNoDeepMerging()
+                                    ->normalizeKeys(false)
+                                    ->useAttributeAsKey('name')
+                                    ->prototype('variable')->end()
+                                ->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('smime_signer')
+                            ->addDefaultsIfNotSet()
+                            ->canBeEnabled()
+                            ->info('SMIME signer configuration')
+                            ->children()
+                                ->scalarNode('key')
+                                    ->info('Path to key (in PEM format)')
+                                    ->defaultValue('')
+                                    ->cannotBeEmpty()
+                                ->end()
+                                ->scalarNode('certificate')
+                                    ->info('Path to certificate (in PEM format without the `file://` prefix)')
+                                    ->defaultValue('')
+                                    ->cannotBeEmpty()
+                                ->end()
+                                ->scalarNode('passphrase')
+                                    ->info('A passphrase of the private key if any')
+                                    ->defaultNull()
+                                ->end()
+                                ->scalarNode('extra_certificates')->defaultNull()->end()
+                                ->integerNode('sign_options')->defaultNull()->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('smime_encrypter')
+                            ->addDefaultsIfNotSet()
+                            ->canBeEnabled()
+                            ->info('SMIME encrypter configuration')
+                            ->children()
+                                ->scalarNode('certificate')
+                                    ->info('Path to certificate (in PEM format without the `file://` prefix)')
+                                    ->defaultValue('')
+                                    ->cannotBeEmpty()
+                                ->end()
+                                ->integerNode('cipher')
+                                    ->info('A set of algorithms used to encrypt the message')
+                                    ->defaultNull()
+                                    ->validate()
+                                    ->ifTrue(function ($v) {
```

Comment:
Please indent the child calls of the `->validate()` until the corresponding `->end()` call which should be back at the same level than `->validate()` (i.e. lines 2264 to 2777)